ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/tree/TomlTable.kt

```diff
@@ -26,6 +28,69 @@ public abstract class TomlTable(
     public abstract var fullTableName: String
     public abstract var tablesList: List<String>
     public abstract val type: TableType
+
+    override fun write(
+        emitter: TomlEmitter,
+        config: TomlConfig,
+        multiline: Boolean
+    ) {
+        // Todo: Option to explicitly define super tables?
+        // Todo: Support dotted key-value pairs (i.e. a.b.c.d = 7)
+
+        val children = children
+
+        val key = TomlKey(fullTableName, 0)
+
+        val firstChild = children.first()
+
+        if (isExplicit(firstChild) && type == TableType.PRIMITIVE) {
+            emitter.writeHeader(key, config)
+
+            if (firstChild !is TomlStubEmptyNode) {
+                emitter.emitNewLine()
```

Comment:
If first child is a stub, this means, that it is a table without children: 
```
[a]
[b]
[c]
```

I suppose we can have a newline after it

Reply:
Yep, that was intentional. We could maybe have some config values to effect formatting stuff like this, utilizing the separate input and output config classes you suggested